packages/manager/modules/pci/src/projects/project/instances/instance/edit/edit.html

```diff
@@ -147,7 +147,7 @@ <h1>{{:: 'pci_projects_project_instances_instance_edit_title' | translate }}</h1
         </form>
     </div>
 
-    <!-- <div class="mt-4 pt-4 border-top">
+    <div class="mt-4 pt-4 border-top">
```

Comment:
`border-top` :(

Looks like we really need a true CSS utility framework

Reply:
No bootstrap4 utils https://getbootstrap.com/docs/4.0/utilities/borders/